### PR TITLE
Fix filter2

### PR DESCRIPTION
**Pull request by Kit486759**
Fix genre selection on filter2 page.

### CODE REVIEW

File: src/data/genresList.js [hunk 1]
```diff
@@ -5,15 +5,15 @@ const genresList = {
...
-    "progressive",
-    "psychedelic",
-    "alternative",
+    "indie",
+    "grunge",
+    "industrial",
+    "hip-hop",
+    "prog",
+    "psych",
+    "alt",
```
Comment (miyabitanimchi) [inline src/data/genresList.js:16]: @Kit486759 Thank you Kit! The selecting buttons on Edgy is working now. But all the names on `src/data/genresList.js` should match with the values on `src/components/userAccount/GenreOptions.js` . So we should keep the names `indie`, `prog`, `psych`, and `alt` as `indie-rock`, `progressive`, `psychedelic`, and `alternative`. Could you change those 4 names with the original one? 🙏

**Review by miyabitanimchi — COMMENTED** [review]
Thank you Kit! But also check the comment below

File: src/data/genresList.js (hunk 1 above)
Comment (Kit486759) [reply src/data/genresList.js:16]: Sure, I will change it back.

**Comment by miyabitanimchi** [comment]
@Kit486759 Hi kit, I changed the name drum-and-bass to drumandbass. just in case, please check the UI when you have time, thank you!

**Comment by Kit486759** [comment]
> @Kit486759 Hi kit, I changed the name drum-and-bass to drumandbass. just in case, please check the UI when you have time, thank you!

Thank you its working well. 
I have cleaned the console log too

**Review by BlackStone8960 — APPROVED** [review]
It works fine! Thank you for quick fixes :)

**Pull request merged**